calico-vpp-agent/routing/connectivity/ipsec.go

```diff
@@ -34,6 +34,29 @@ type IpsecProvider struct {
 }
 
 func (p *IpsecProvider) OnVppRestart() {
+	var err error
+	err = p.vpp.SetIPsecAsyncMode(true)
```

Comment:
Maybe we should do this only if `config.IpsecNbAsyncCryptoThread > 0` so that we can also test with async crypto off ?